parts/linux/cloud-init/artifacts/ipv6_nftables

```diff
@@ -0,0 +1,15 @@
+#!/sbin/nft -f
```

Comment:
does this actually work? you are directly executing with bash in the system oneshot, I would expect bash to parse this as a shell script and fail?

Reply:
oo nvm I see how it works, yeah would drop the shebang if this isn't a real script 

I see it seems to be a convention, albeit a dated (?) one: https://bugzilla.redhat.com/show_bug.cgi?id=1756347

up to you if you want to leave it

Reply:
not sure if this comment was on the latest commit but right now:

- service has `ExecStart=/bin/bash /opt/scripts/ipv6_nftables.sh`
- then that script has `nft -f $NFTABLES_RULESET_FILE`
- and the nftables rule file still has `#!/sbin/nft -f`

this all seems to work. However, I noticed that when installing nft on a node it gets installed to `/usr/sbin` so should I just remove the `#!/sbin/nft -f` from the rule file altogether?

Reply:
yep sorry, I misread the ExecStart as directly executing the table, missed the misdirection with the IMDS bits initially .

Reply:
👍  ... disregard my comment then, I wrote it before the page updated with your second comment. I'll just remove the shebang from ipv6_nftables file